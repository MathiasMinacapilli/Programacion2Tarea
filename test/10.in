# pruebas de mayores 2
mayores 4 4 (9,a) (-1,b) (3,c) (9,d) (5,e) (1,f) (11,g) (10,h) (4,i) (2,j) .
mayores 4 4 (9,a) (-1,b) (13,c) (9,d) (5,e) (12,f) (11,g) (10,h) (4,i) (2,j) .
mayores 10 10 (9,a) (1,b) (2,c) (6,d) (5,e) (10,f) (7,g) (3,h) (4,i) (8,j) .
mayores 10 9 (9,a) (1,b) (2,c) (6,d) (5,e) (10,f) (7,g) (3,h) (4,i) (8,j) .
mayores 9 10 (9,a) (1,b) (2,c) (6,d) (5,e) (10,f) (7,g) (3,h) (4,i) (8,j) .
mayores 3 10 (1,a) (2,b) (3,c) (4,d) (5,e) (6,f) (7,g) (8,h) (9,i) (10,j) .
mayores 3 7 (1,a) (2,b) (3,c) (4,d) (5,e) (6,f) (7,g) (8,h) (9,i) (10,j) .
mayores 3 4 (1,a) (2,b) (3,c) (4,d) (5,e) (6,f) (7,g) (8,h) (9,i) (10,j) .
mayores 1 1 (1,a) (2,b) (3,c) (4,d) (5,e) (6,f) (7,g) (8,h) (9,i) (10,j) .
mayores 4 4 (9,a) (9,z) (-1,b) (3,c) (3,t) (9,d) (5,e) (1,a) (1,f) (11,g) (10,h) (4,m) (4,i) (2,j) .
mayores 10 10 (9,z) (9,a) (2,h) (1,b) (2,c) (6,d) (5,e) (10,f) (10,k) (7,g) (3,t) (3,h) (4,i) (8,j) .
mayores 1 1 (1,a) (2,b) (3,c) (4,d) (5,e) (6,f) (7,g) (8,h) (9,i) (1,m) (10,j) .
mayores 1 10 (1,a) (2,b) (10,m) (3,c) (4,d) (5,e) (6,f) (7,g) (8,h) (9,i) (10,j) .
mayores 1 10 (1,a) (2,b) (3,c) (4,d) (5,e) (6,f) (7,g) (8,h) (9,i) (10,j) (10,m).
Fin
