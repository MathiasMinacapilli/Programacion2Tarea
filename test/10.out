1>#  pruebas de mayores 2.
2>(1,f)(2,j)(3,c)(4,i)
3>(2,j)(4,i)
4>(1,b)(2,c)(3,h)(4,i)(5,e)(6,d)(7,g)(8,j)(9,a)(10,f)
5>(1,b)(2,c)(3,h)(4,i)(5,e)(6,d)(7,g)(8,j)(9,a)
6>(2,c)(3,h)(4,i)(5,e)(6,d)(7,g)(8,j)(9,a)(10,f)
7>(8,h)(9,i)(10,j)
8>(5,e)(6,f)(7,g)
9>(2,b)(3,c)(4,d)
10>(1,a)
11>(1,a)(2,j)(3,c)(4,m)
12>(1,b)(2,h)(3,t)(4,i)(5,e)(6,d)(7,g)(8,j)(9,z)(10,f)
13>(1,a)
14>(10,m)
15>(10,j)
16>Fin.
